<?xml version="1.0"?>

<launch>
    
    <node pkg="points_process_test" type="points_process_test_node" name="points_process_test_node" output="screen">
        
        <param name="sub_topic" value="/pandar_points"/>
        <param name="pub_topic" value="/pandar_points_processed"/>
        
        <param name="is_limit_mode" value="true"/>
        <param name="is_clip_mode" value="true"/>
        <param name="is_filter_mode" value="true"/>
        <param name="is_downsample_mode" value="true"/>
        <param name="is_show_points_size" value="true"/>
        
        <param name="the_field_of_view" value="45"/>

        <param name="the_sensor_height" value="1.7"/>
        <param name="the_view_higher_limit" value="2.0"/>
        <param name="the_view_lower_limit" value="-1.0"/>
        <param name="the_min_distance" value="2.0"/>
        <param name="the_max_distance" value="100.0"/>
        
        <param name="the_meank" value="10"/>
        <param name="the_stdmul" value="1.0"/>
        <param name="the_leafsize" value="0.1"/>
        
    </node>

</launch>
